test/vanilla/Expected/AcceptanceTests/NonStringEnums/nonstringenums/models/_non_string_enums_client_enums.py

```diff
@@ -22,7 +22,7 @@ class FloatEnum(float, Enum):
     """List of float enums
     """
 
-    two_hundred4 = 200.4
+    two_hundred = 200
```

Comment:
I don't think that was generated with testserver .26, since I changed this to 200.4 in that version